lib/ts-types/js-yaml.d.ts

```diff
@@ -0,0 +1,165 @@
+// Type definitions for js-yaml v4.0.0
+// Project: https://github.com/nodeca/js-yaml
+
+// Adapted from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/65afafadd93759348909de1cdad5df5789c94326/types/js-yaml/index.d.ts
+// 1. fixed compatibility with TypeScript 2 (which we're still
+//    using at the time of writing) by replacing `unknown` with `any`
```

Comment:
Shall we create a separate task to remind ourselves or whoever that would be that we'll benefit from upgrading version of TypeScript + we can remove this hack?

Reply:
See https://github.com/kaitai-io/kaitai_struct_webide/issues/167